src/pages/projectSettings/tabs/ViewsAndFiltersTab/transformers.ts

```diff
@@ -1,40 +1,45 @@
 import { ProjectSettingsTabRoutes } from "constants/routes";
-import { ProjectSettingsQuery } from "gql/generated/types";
 import { FormToGqlFunction, GqlToFormFunction } from "../types";
+import { ProjectType } from "../utils";
 
 type Tab = ProjectSettingsTabRoutes.ViewsAndFilters;
 
-export const gqlToForm = ((data: ProjectSettingsQuery["projectSettings"]) => {
+export const gqlToForm = ((data, { projectType }) => {
   if (!data) return null;
 
-  const {
-    projectRef: { parsleyFilters, projectHealthView },
-  } = data;
+  const { projectRef } = data;
 
   return {
     parsleyFilters:
-      parsleyFilters?.map(({ caseSensitive, exactMatch, expression }) => ({
-        displayTitle: expression,
-        expression,
-        caseSensitive,
-        exactMatch,
-      })) ?? [],
-    view: {
-      projectHealthView,
-    },
+      projectRef.parsleyFilters?.map(
+        ({ caseSensitive, exactMatch, expression }) => ({
+          displayTitle: expression,
+          expression,
+          caseSensitive,
+          exactMatch,
+        })
+      ) ?? [],
+    ...(projectType !== ProjectType.Repo &&
+      "projectHealthView" in projectRef && {
```

Comment:
Is it necessary to check the `ProjectType`?
```suggestion
    ...("projectHealthView" in projectRef && {
```

Reply:
Not strictly necessary, but this is the pattern we've used on other pages for non-repo settings and I feel like it makes it clear that that's the intended purpose.